src/features/terraformCloud.ts

```diff
@@ -74,6 +74,8 @@ export class TerraformCloudFeature implements vscode.Disposable {
       showCollapseAll: true,
       treeDataProvider: workspaceDataProvider,
     });
+    const organization = this.context.workspaceState.get('terraform.cloud.organization', '');
+    workspaceView.title = organization !== '' ? `Workspaces - [${organization}]` : 'Workspaces';
```

Comment:
```suggestion
    workspaceView.title = organization !== '' ? `Workspaces - (${organization})` : 'Workspaces';
```

Further down, we use `( )` for the title

Reply:
Yes, we do, but we use `[ ] ` for denoting Projects right underneath this title, so I thought matching that was more important visually.

I was hesitant about using any bracket type thing at all in the statusbar because its not common to do that, so I'd be inclined to remove the `( )` there before changing the title.

Reply:
I don't have any strong opinions here, as long as we keep things consistent.

Right now we use `[ ]` on first load
<img width="377" alt="CleanShot 2023-10-13 at 14 19 47@2x" src="https://github.com/hashicorp/vscode-terraform/assets/45985/623aaa49-ce86-4e12-b38c-15d2b84b5f71">

and `( )` after changing the org
<img width="385" alt="CleanShot 2023-10-13 at 14 19 59@2x" src="https://github.com/hashicorp/vscode-terraform/assets/45985/be0b6d78-430b-4c0c-aba9-2048d27e95dd">

---

VS Code uses `( )`for workspaces, but I think it would also work without any brackets
<img width="380" alt="CleanShot 2023-10-13 at 14 21 09@2x" src="https://github.com/hashicorp/vscode-terraform/assets/45985/2c9e2f85-2d1f-4cf1-a10b-f8fd57adbd2f">